### PR TITLE
build: add aarch64 wheel build support

### PR DESCRIPTION
**Pull request by odidev**
Added aarch64 wheel build support.
Related to https://github.com/deepcharles/ruptures/issues/179, @julia-shenshina Could you please review this PR?

### CODE REVIEW

File: .github/workflows/upload-to-pypi.yml [hunk 1]
```diff
@@ -73,8 +73,35 @@ jobs:
...
+      matrix:
+        python: [36, 37, 38, 39]
+        include:
+          - os: ubuntu-latest
+            arch: aarch64
+            platform_id: manylinux_aarch64
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Set up QEMU
```
Comment (oboulant) [inline .github/workflows/upload-to-pypi.yml:89]: What does it do ? It helps with cross-compilation on aarch64 ?

Comment (odidev) [reply .github/workflows/upload-to-pypi.yml:89]: It is used for setting up QEMU for enabling AArch64 binary execution on x86 machine.

**Comment by deepcharles** [comment]
Looks good to me, even though I cannot test it before it is merged. Many thanks!

**Comment by oboulant** [comment]
> Looks good to me, even though I cannot test it before it is merged. Many thanks!

If you manually run the Gh Actions on this specific branch and labelling it as a release candidate, it is not possible to test it ?

**Comment by deepcharles** [comment]
> > Looks good to me, even though I cannot test it before it is merged. Many thanks!
> 
> If you manually run the Gh Actions on this specific branch and labelling it as a release candidate, it is not possible to test it ?

I can only run the GH Action on a branch from this repo, but not on a branch from a fork.

**Comment by oboulant** [comment]
> > > Looks good to me, even though I cannot test it before it is merged. Many thanks!
> > 
> > 
> > If you manually run the Gh Actions on this specific branch and labelling it as a release candidate, it is not possible to test it ?
> 
> I can only run the GH Action on a branch from this repo, but not on a branch from a fork.

In that case, maybe @odidev can modify the PR to be merge on a separate branch on `ruptures` repo. Then we can test, right ?

**Comment by odidev** [comment]
Following are the logs for wheel build and test. Please have a look and let me know if any changes are required in the PR:

1. Wheel build: https://github.com/odidev/ruptures/actions/runs/1036426161 
	
2. Test logs: https://github.com/odidev/ruptures/actions/runs/1036426160

**Comment by oboulant** [comment]
Thanks for the links !
Looks good to me, thx !

**Comment by deepcharles** [comment]
Looks good to me, manyy thanks.

**Pull request merged**